src/rebar_file_utils.erl

```diff
@@ -120,18 +120,13 @@ mv(Source, Dest) ->
                                       [{use_stdout, false}, abort_on_error]),
             ok;
         {win32, _} ->
-            {ok, R} = rebar_utils:sh(
-                        ?FMT("move /y \"~s\" \"~s\" 1> nul",
-                             [filename:nativename(Source),
-                              filename:nativename(Dest)]),
-                        [{use_stdout, false}, return_on_error]),
-            case R of
-                [] ->
-                    ok;
-                _ ->
-                    {error, lists:flatten(
-                              io_lib:format("Failed to move ~s to ~s~n",
-                                            [Source, Dest]))}
+            case file:rename(Source, Dest) of
```

Comment:
I use robocopy on cp_r but not move. Can try to look into that and see if I get any success.